MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+module(
+    name = "rules_avro",
+    version = "0.1.0",
```

Comment:
Not sure whether this is a reasonable version.

Reply:
@honnix does it mean, I also need to tag the commit?

Reply:
I don't exactly know how to bump the version in BCR since I have never done that. We can follow that up for sure.

Reply:
Hi @chenrui333 or @honnix  , do you mind creating an issue for when the ruleset will be available in BCR? I'd like something to track so I can get notified when to update. Thanks.

Reply:
#57